packages/containerd.rb

```diff
@@ -0,0 +1,160 @@
+# Adapted from Arch Linux containerd PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/containerd/trunk/PKGBUILD
+
+require 'package'
+
+class Containerd < Package
+  description 'An open and reliable container runtime'
+  homepage 'https://containerd.io/'
+  version '1.6.15'
+  compatibility 'all'
+  license 'Apache'
+  source_url 'https://github.com/containerd/containerd.git'
+  git_hashtag "v#{version}"
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_armv7l/containerd-1.6.1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_armv7l/containerd-1.6.1-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/containerd/1.6.1_x86_64/containerd-1.6.1-chromeos-x86_64.tar.zst'
```

Comment:
Why do binaries don't match the version?

Reply:
Ah! good catch. My script got confused with the version number since there is a "Version = 2" in the file...

rebuilding now...

Reply:
Fixed.